src/result/error.rs

```diff
@@ -40,21 +40,21 @@ impl Error {
         Self { code, info }
     }
 
-    // Use internally to create an Error object without error info. Typically used with QueryInterface
-    // (E_NOINTERFACE) or the absense of an object to return (E_POINTER) to avoid the code gen overhead
-    // for casts that should be cheap (few instructions). Think of it as a way to create recoverable errors
-    // that don't need th overhead of debugging origination info.
-    pub fn fast_error(code: HRESULT) -> Self {
+    /// Use internally to create an Error object without error info. Typically used with [`Interface::query`]
```

Comment:
```suggestion
    /// Used internally to create an Error object without error info. Typically used with [`Interface::query`]
```

Reply:
I don't think this mention makes much sense anyway as the function is public.

Reply:
Yes, `fast_error` should just be hidden from docs.

Reply:
Added `#[doc(hidden)]` while still keeping it as doc-comment for internal development purposes like what rust-analyzer is capable of displaying.

Reply:
I don't see the `doc(hidden` attribute. Missing commit?

Reply:
I pushed it into #855 only, by accident. Now it should be here, too.